packages/plugins/Calendar/src/hooks/useEventList.tsx

```diff
@@ -0,0 +1,50 @@
+import { useQuery } from '@tanstack/react-query'
+import { Calendar } from '@masknet/web3-providers'
+
+export function useNewsList() {
+    const date = Math.floor(Date.now() / 1000)
+    const { data, isLoading } = useQuery<any>(
+        ['newsList', Math.floor(date / 1000)],
```

Comment:
Divide 1000 twice.

Reply:
Need to divide twice to get a stable query key